src/bio/bio_xstream.c

```diff
@@ -242,7 +242,7 @@ populate_whitelist(struct spdk_env_opts *opts)
 		return -DER_NOMEM;
 
 	for (i = 0; i < DAOS_NVME_MAX_CTRLRS; i++) {
-		memset(trid, 0, sizeof(struct spdk_nvme_transport_id));
+		memset(trid, 0, sizeof(*trid));
```

Comment:
technically, this could be moved to the end of the loop since D_ALLOC_PTR clears the memory initially

Reply:
Agree.